src/main/drivers/gyro_sync.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * gyro_sync.h
+ *
+ *  Created on: 3 aug. 2015
+ *      Author: borisb
+ */
+
+extern bool gyroIsUpdated;
```

Comment:
IMO returning bool from check function is cleaner  (caching may be done internally if necessary)